cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -533,7 +533,7 @@ initLiveViewState = do
                 , lvsNetworkUsageOutLast    = 0
                 , lvsNetworkUsageOutNs      = 10000
                 , lvsMempoolCapacity        = mempoolCapacity
-                , lvsMempoolCapacityBytes   = mempoolCapacity * maxBytesPerTx
+                , lvsMempoolCapacityBytes   = mempoolCapacity * maxBytesPerTx -- / 1024
```

Comment:
@ksaric, do we need this comment here?

Reply:
It's going to be fixed in the next PR, probably should have added `TODO(ks): ` before that.